DSA/Double_Hashing.java

```diff
@@ -0,0 +1,166 @@
+import java.util.*;
+
+public class Double_Hashing {
+    int m,n,PM;
```

Comment:
Use better variable names. This will improve the readability of code.

Reply:
Okay sir